packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -402,11 +417,10 @@ export class WorkflowsController {
 		);
 	}
 
+	@Licensed('feat:sharing')
```

Comment:
@despairblue it fails because of this change
Isn't it doing the same?
If I roll it back it works

Reply:
This was correct. I don't know why the tests passed at all. The tests are in the none-ee file. The requests should rightfully be 404s not 200s.

The tests should be in: `packages/cli/test/integration/workflows/workflows.controller.ee.test.ts`, or actually even in the service test file, since they test service logic, not controller logic.